ddspipe_core/include/ddspipe_core/efficiency/payload/PayloadPoolMediator.hpp

```diff
@@ -115,14 +113,14 @@ class PayloadPoolMediator : public fastrtps::rtps::IPayloadPool
      * @param data the data to be written by the \c writer.
      * @param handle the data to be used by the \c writer.
      *
-     * @return true if everything OK
-     * @return false if something went wrong
+     * @return ReturnCode::PRECONDITION_NOT_MET if the handle introduced does not match with the one associated to the
+     * data, ReturnCode::OK if the data is correctly sent and ReturnCode::ERROR otherwise.
```

Comment:
Why this involved explanation? I this is Fast-DDS API behaviour I don't think should be explained here.

Reply:
If we're transmitting the ReturnCodes, I don't see a reason not to explicitly indicate what each return value means.

Reply:
So I just leave the possible ReturnCodes without explanation?

Reply:
I guess that would be the correct option yes.